multiqc/modules/pangolin/pangolin.py

```diff
@@ -235,3 +245,23 @@ def pangolin_table(self):
         }
 
         return table.plot(self.pangolin_data, headers, table_config)
+
+
+def _format_qc_notes(raw: str) -> Optional[str]:
+    """
+    Parses QC notes, they appear to come from:
+    https://github.com/cov-lineages/pangolin/blob/361f49cbffbf26eb28bed2f4a4c0e7f3d5a054cc/pangolin/utils/preprocessing.py#L91-L97
+    https://github.com/cov-lineages/pangolin/blob/361f49cbffbf26eb28bed2f4a4c0e7f3d5a054cc/pangolin/utils/preprocessing.py#L179
+    """
+    if raw.startswith("Ambiguous_content"):
+        # e.g. Ambiguous_content:0.03
+        split = raw.split(":")
+        if len(split) != 2:
+            logging.warning(f"Expected label of format 'Ambiguous_content:0.01', found: '{raw}'")
+            return None
```

Comment:
I replaced the assertion with a warning, so the module wouldn't fail completely if one input file is off.